autoprotocol/container.py

```diff
@@ -71,7 +71,7 @@ def set_properties(self, properties):
         self.properties = properties.copy()
         return self
 
-    def add_properties(self, properties):
+    def add_properties(self, properties, silence_warnings=False):
```

Comment:
I would make this more specific to the action. Something along the lines of "overwrite_existing_keys"